ledger/src/blockstore_db.rs

```diff
@@ -291,65 +291,33 @@ struct Rocks(rocksdb::DB, ActualAccessType, OldestSlot);
 
 impl Rocks {
     fn open(path: &Path, options: BlockstoreOptions) -> Result<Rocks> {
-        use columns::*;
-        let access_type = options.access_type;
-        let recovery_mode = options.recovery_mode;
+        let access_type = &options.access_type;
+        let recovery_mode = options.recovery_mode.clone();
```

Comment:
Did the borrow / copy to avoid partial move so I could pass the entire `options` into `Self:: cf_descriptors()` later